Gopkg.toml

```diff
@@ -109,3 +109,7 @@
 [prune]
   go-tests = true
   unused-packages = true
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/ExchangeUnion/swap-resolver"
```

Comment:
We're actively making efforts to _reduce_ the set of dependancies we pull into the project. Why can't this be added as a _purely_ RPC feature?

Reply:
Resolved by latest commit. 
Dependency removed.